applications/add_file_to_db.py

```diff
@@ -61,10 +60,10 @@ def _userConfirm():
 
 def main():
 
-    db = db_handler.DatabaseHandler()
-
-    parser = argparser.CommandLineParser(description=("Add a file or files to the DB."))
-    group = parser.add_mutually_exclusive_group(required=True)
+    # temporary solution; class values in db_handler
+    _tmp_db_config = db_handler.DatabaseHandler()
```

Comment:
All what we need are the values defined here: https://github.com/gammasim/gammasim-tools/blob/c575aa5088da0d3d3eeb3c00fd7a501725c20576/simtools/db_handler.py#L59

There is a TODO above these lines, so expect that you want to do something with it. We don't need a DB connection, it is really only about getting e.g., `DB_TABULATED_DAT`.

Reply:
That TODO is old and was before we decided to remove the config file. At the moment I think it's better to actually have these definitions where they are. So I think the best solution is to allow initialising the db_handler without the connection info, so you can have these values without an actual DB connection. Then allow setting up the DB details the original init with a separate function. That function would also open the connection. What do you think? I can implement it in this branch so it can be part of this PR.

Reply:
Yes, I think this would be the best solution. Could you implement it?
(I didn't want to touch the db_handler, as I don't understand everything)

Reply:
This is implemented now but an issue with multiple parsing was found while testing it (discussed separately on mattermost).